src/mmw/js/src/analyze/controllers.js

```diff
@@ -18,6 +18,11 @@ var AnalyzeController = {
         if (!App.map.get('maskLayerApplied')) {
             App.map.set('maskLayerApplied', true);
         }
+
+        // Clear currProject so that
+        // the ModelingController will know to make a
+        // new project when coming from the Analyze views.
+        App.currProject = null;
```

Comment:
Thanks, it would be great if you could evaluate this from the standpoint of ITSI compatibility.